.github/workflows/changelog_generation.yaml

```diff
@@ -22,7 +22,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
-        fetch-depth: 0
```

Comment:
The fetch-depth default 1 seems enough. No need to fetch all branches.
https://github.com/actions/checkout#checkout-a-different-branch